aptos-move/framework/move-stdlib/sources/configs/config_for_next_epoch.move

```diff
@@ -0,0 +1,78 @@
+/// This wrapper helps store an on-chain config for the next epoch.
+///
+/// Once reconfigure with DKG is introduced, every on-chain config `C` should do the following.
+/// - Support async update when DKG is enabled. This is typically done by 3 steps below.
+///   - Implement `C::set_for_next_epoch()` using `upsert()` function in this module.
+///   - Implement `C::on_new_epoch()` using `extract()` function in this module.
+///   - Update `0x1::reconfiguration_with_dkg::finish()` to call `C::on_new_epoch()`.
+/// - Support sychronous update when DKG is disabled.
+///   This is typically done by implementing `C::set()` to update the config resource directly.
+///
+/// NOTE: on-chain config `0x1::state::ValidatorSet` implemented its own buffer.
+module std::config_for_next_epoch {
```

Comment:
Call it `buffered_config_for_next_epoch` or `buffered_config` ?

Reply:
IMO it's a config *buffer, wrapper, or holder*...